sympy/stats/rv.py

```diff
@@ -1265,19 +1286,21 @@ def given_fn_subs(*args):
     else:
         fn = lambdify(rvs, expr, modules=library, **kwargs)
 
+
     if condition is not None:
         given_fn = lambdify(rvs, condition, **kwargs)
 
     def return_generator_infinite():
         count = 0
         np = import_module('numpy')
         if np:
-            rand_state = np.random.default_rng(seed=seed)
+            rand_state = np.random.default_rng(seed=seed) if library != "pymc3" else seed
         else:
             rand_state = None
+        _size = (1,)+((size,) if isinstance(size, int) else size)
```

Comment:
Why is the `1` being added here?

Reply:
Some sampling methods like in matrix distributions and joint rv require both numsamples and size. Since numsamples for return_generator_infinte is effectively 1 as it returns one sample only at one time.So, I added 1.